README.md

```diff
@@ -16,8 +16,10 @@ it.
 XHGui has the following requirements:
 
 - PHP version 7.0 up to 7.3
-- If using MongoDB storage, see [MongoDB](#MongoDB) requirements
-- If using PDO storage, see [PDO](#PDO) requirements
+- [MongoDB Extension][ext-mongodb] MongoDB PHP driver: `pecl install mongodb`
+  XHGui requires verison 1.3.0 or later.
```

Comment:
why expand mongo deps here? they are listed in detail in mongodb sub-section.

Reply:
Because that implies they are only necessary if you are using MongoDB, which would be a lie

Reply:
Since the PDO section mentions the MongoDB extension explicitly, I've eliminated that line